### PR TITLE
Expose e10s status in panel

### PR DESCRIPTION
**Pull request by wagnerand**

### CODE REVIEW

**Comment by wagnerand** [comment]
screen shot 2016-07-26 at 18 53 56

I know the icon is broken but that is not trivial to fix.

File: data/reporter.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ var initialize = function(data) {
         version.appendChild(document.createTextNode("Version "));
         version.appendChild(document.createTextNode(document.addonReports[i].version));
         td2.appendChild(version);
+
+        var e10s = document.createElement("div");
+        e10s.setAttribute("class", "version");
+        e10s.appendChild(document.createTextNode((!document.addonReports[i].multiprocessCompatible ? "Not compatible" : "Compatible") + " to multi-process."));
```
Comment (tofumatt) [inline data/reporter.js]: "with multiprocess" instead "to multiprocess" reads better I think.

Also if this is just for us/add-on developers, we can probably just say e10s.

**Comment by brianking** [comment]
The messaging is targeted at users.

I'm wondering if even saying multi-process will be enough? Will users even know what that is? Can we add a More... link that brings them out somewhere that explains more?

**Comment by wagnerand** [comment]
@brianking Happy to add a "More" link. What document should we link to?

**Comment by wagnerand** [comment]
@brianking Due to server-side changes I have to release a new version of ACR tomorrow. Please let me know asap.

If it's not that pressing, we can merge as is and fix it in an update we ship later.

**Comment by wagnerand** [comment]
Since tofumatt is out, could you give this a look, please @kewisch? Thanks!

File: data/reporter.htm [hunk 2]
```diff
@@ -22,6 +22,7 @@
     </div>
 
     <div id="buttons">
+        <span id="appe10senabled"></span>
```
Comment (kewisch) [inline data/reporter.htm]: I'd sugget to use camelCase or whatever-the-dashes-case-is-called here for consistency

File: data/reporter.js [hunk 3]
```diff
@@ -50,6 +50,11 @@ var initialize = function(data) {
         version.appendChild(document.createTextNode("Version "));
         version.appendChild(document.createTextNode(document.addonReports[i].version));
         td2.appendChild(version);
+
+        var e10s = document.createElement("div");
+        e10s.setAttribute("class", "version");
+        e10s.appendChild(document.createTextNode((!document.addonReports[i].multiprocessCompatible ? "Not compatible" : "Compatible") + " with multi-process."));
```
Comment (kewisch) [inline data/reporter.js]: No need to negate this, you can just flip the if/else parts.

File: data/reporter.js [hunk 4]
```diff
@@ -93,6 +98,10 @@ var initialize = function(data) {
     document.initialized = true;
 };
 
+self.port.on("app_e10s_enabled", function(enabled) {
+    $("#appe10senabled").text("Multi-process is " + (!enabled ? "not " : "" ) + "enabled.");
```
Comment (kewisch) [inline data/reporter.js]: Same here

**Comment by kewisch** [comment]
@wagnerand r+ with comments considered

**Comment by brianking** [comment]
@wagnerand I thought there might be a wiki page explaining compatibility. But I can't find one. We should consider throwing one up, even on addons.mozilla.org for more legitimacy. Although the wiki as first pass would be fine.

Sure, of to add it in later.

**Comment by wagnerand** [comment]
@brianking it might just be a matter of taking some content from
* https://developer.mozilla.org/en-US/Firefox/Multiprocess_Firefox (intro text)
* https://developer.mozilla.org/en-US/Firefox/Multiprocess_Firefox/Motivation

Do you know who could help writing that page? Also adding @atsay here.

**Comment by wagnerand** [comment]
Follow-up in https://github.com/mozilla/ACR/issues/34

**Comment by atsay** [comment]
If by "users", you're talking about add-on developers, there is already a ton of pages about compatibility you can point people to. Like this: https://developer.mozilla.org/en-US/Add-ons/Working_with_multiprocess_Firefox

If you're talking about end users, you can point them here: https://wiki.mozilla.org/Firefox/multiprocess

**Pull request merged**